### ISSUE TITLE
Update 

### ISSUE THREAD

**Issue by FITFC** [impression28/LMC-markets#3]
Hello, could you provide an already compiled version?

**Comment by rigille** [comment]
Sure I can but I need some time to do so. Do you want to make a similar site for your server? I would like to know more about your use case

**Comment by ghost** [comment]
Do you compile javascript html and stuff? Is'nt it all client side just asking

**Comment by rigille** [comment]
yeah it's all client side, the only thing the server does is serving static files. being more specific I made python scripts that parse the ChestShop log and create both tsv files with transaction data and an html file with a table for the main page. the graph page is made with some javascript that GETs the corresponding tsv file and plots it with plotly. not an industry standard by any means hahha
